### PR TITLE
Add HEOS media player component docs

### PR DESCRIPTION
**Pull request by easink**
**Description:**

Denon HEOS speaker.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#21721

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/media_player.heos.page [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    name: HEOS Player
+    host: HEOS-1
+```
+
+{% configuration %}
+host:
+  description: "Address of the device. Example: 192.168.1.32"
+  required: false
+  type: string
+name:
```
Comment (klaasnicolaas) [inline source/_components/media_player.heos.page]: missing the default

File: source/_components/media_player.heos.page [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+To add a Denon HEOS speaker to your installation, add the following to your `configuration.yaml` file:
+
+## {% linkable_title HEOS platform %}
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: heos
+    name: HEOS Player
```
Comment (klaasnicolaas) [inline source/_components/media_player.heos.page]: You can remove this line

File: source/_components/media_player.heos.page [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    host: HEOS-1
```
Comment (klaasnicolaas) [inline source/_components/media_player.heos.page]:
```suggestion
    host: IP_ADDRESS
```

File: source/_components/media_player.heos.page [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+layout: page
+title: "Denon HEOS speakers"
+description: "Instructions on how to integrate Denon HEOS into Home Assistant."
+date: 2019-03-06 22:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Media Player
+---
```
Comment (klaasnicolaas) [inline source/_components/media_player.heos.page]: missing `ha_release` and a logo

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/media_player.heos.page [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+title: "Denon HEOS speakers"
+description: "Instructions on how to integrate Denon HEOS into Home Assistant."
+date: 2019-03-06 22:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: heos.png
+ha_category: Media Player
+ha_release: "0.90"
```
Comment (klaasnicolaas) [inline source/_components/media_player.heos.page]:
```suggestion
ha_release: 0.91
```

File: source/_components/media_player.heos.page [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+# Example configuration.yaml entry
+media_player:
+  - platform: heos
+    name: HEOS Player
+    host: IP_ADDRESS
+```
+
+{% configuration %}
+host:
+  description: "Address of the device. Example: 192.168.1.32".
```
Comment (klaasnicolaas) [inline source/_components/media_player.heos.page]:
```suggestion
  description: "Address of the device. Example: 192.168.1.32."
```

File: source/_components/media_player.heos.page [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+  required: false
+  type: string
+{% endconfiguration %}
+
+A few notes for platform: heos
+
+- Does not handle multiple speakers by groups.
+- Seeking is not implemented at this moment.
+
+[HEOS]: /components/media_player.heos/
```
Comment (klaasnicolaas) [inline source/_components/media_player.heos.page]: Are you using this link? I see no reference to it

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/media_player.heos.page (hunk 7 above)
Comment (easink) [reply source/_components/media_player.heos.page]: nope, removed

**Review by klaasnicolaas — APPROVED** [review]
✅ Can be merged after merging the parent PR 👍

**Pull request merged**